### ISSUE TITLE
Feature Request: add unit tests

### ISSUE THREAD

**Issue by HunteRoi** [imranbarbhuiya/pagination.djs#6]
Unit testing this library would be a great addition for both you, your maintainers and people using the NPM package.

I recommend looking into [Jest](https://jestjs.io/) as it is well documented and also open-source.

**Comment by imranbarbhuiya** [comment]
Yeah I've plan to add these tests. But exam is going on so I'm not able to work  I'll be adding them after my exam.
